src/pages/Precaution/Interfaces/UserPrecauctionTableInfo.tsx

```diff
@@ -1,3 +1,5 @@
+import { UserPrecautionStatus } from "./UserPrecaution";
+
 type UserPrecautionTableInfo = {
```

Comment:
If I understood correctly, you have to models(`UserPrecautionTableInfo ` and `UserPrecaution`) with same properties, you can implement one from another and have common properties

Reply:
UserPrecautionTableInfo it`s like ViewModel at this situation. UserPrecaution it`s like entity for connection with api.
UserPrecautionTableInfo has computed properties.